aten/src/ATen/core/Type.h

```diff
@@ -83,6 +83,8 @@ enum class TypeID {
   SparseCUDALong,
   SparseCUDAShort,
   SparseCUDAQInt8,
+  AffineCPUQInt8,
```

Comment:
What's the difference between `CPUQInt8` and `AffineCPUQInt8`?

Reply:
QInt8 is the data type and CPU/AffineCPU is the backend. AffineCPU means it is quantized using Affine quantization on CPU device.

Reply:
Then what quantization method is used if the tensor is `CPUQInt8`?

Reply:
This is actually not supported. QInt8 only works for quantized TensorTypeId

Reply:
Oh, then that makes sense. I saw tensor type `CPUQInt8` is there too together with other `XYZQInt8` tensor types. Shall we remove them in the future?

Reply:
these are generated...we can guard against these types for now